deploy/template.yaml

```diff
@@ -48,6 +48,9 @@ Resources:
     Properties:
       CodeUri: ../dist/session.zip
```

Comment:
I suppose if there was some test, I could have approve this and the enhancement I suggested could have come letter. Since this does seen like it would write the session-id to the table

Reply:
The zip method actually allows for two things:
1. It allows use of a shared library in the project between lambdas
2. It shrinks file sizes to be stored in deployment considerably (eg: 29MB vs 89MB)